src/Installer/redist-installer/targets/GenerateMSIs.targets

```diff
@@ -417,7 +416,6 @@
                             GenerateSdkPlaceholderMsi;
                             GenerateToolsetNupkg;
                             GenerateTemplatesNupkgs;
-                            GenerateSdkPlaceholderNupkg"
-        Condition=" '$(OS)' == 'Windows_NT' and '$(Architecture)' != 'arm' " />
```

Comment:
Just curious how is Windows-only condition for MSIs handled now. I see that these conditions were removed from all MSI-related targets.

Reply:
src/Installer/redist-installer/Directory.Build.targets now conditions the GenerateMSIs.targets import.